web/static/svg/close.svg

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
You might need to add `pipenv run python manage.py collectstatic --no-input` to the release script